client/src/pages/Home.js

```diff
@@ -15,12 +15,13 @@ const FlexChild = styled.div`
   margin-bottom: 2rem;
 `;
 
-const StyledIntro = styled.div`
+const StyledIntro = styled.div` 
+
   @media screen and ${mq.phone.narrow.max} {
```

Comment:
Hi @TenNga I'll add my review as I'm trying to help with getting PR's merged. I'd echo @anniegiang comments on consistent styling. To add the margin between the description and links should be consistent(ish) w phone.narrow.
```suggestion
  @media screen and ${mq.phone.wide.max} {
```
If you think `4rem` is too much as it pushes the first button too much down the screen then at least `2rem`. But I don't think that's an issue keeping them the same is probably best.

Reply:
I used phone wide cause I need to show a mobile version when the screen is under 767px. and the phone narrow doesn't have option for 767px.

Reply:
Thank you for your review @joshmorel. I totally agree with you and now I will change narrow to wide right away